index.test-d.ts

```diff
@@ -31,7 +31,7 @@ const hashMap = {
 	foo: 'bar'
 };
 
-expectType<Promise<{[key in 'unicorn' | 'foo']: string | number}>>(
+expectType<Promise<{ unicon: 1, foo: 'bar' }>>(
```

Comment:
```suggestion
expectType<Promise<{ unicorn: 1, foo: 'bar' }>>(
```

Reply:
Good catch, thanks!

I also aligned the whitespace (0966790) and interface field delimiter `;` (8b22075), so that it's consistent with the rest of the file.